docs/modules/deployment/pages/core/getting-started.adoc

```diff
@@ -125,6 +125,33 @@ include::docker/initialize.adoc[]
 endif::[]
 ====
 
+== Receiving SNMP Traps/Informs
```

Comment:
```suggestion
== Receive SNMP Traps/Informs
```

Reply:
This is a NH style thing. They suggest not using "ing" verb forms when possible.